src/app/wizard/step/provider-settings/provider/basic/aws/component.ts

```diff
@@ -117,6 +117,14 @@ export class AWSProviderBasicComponent extends BaseFormValidator implements OnIn
       .pipe(takeUntil(this._unsubscribe))
       .subscribe(_ => this.form.reset());
 
+    this._clusterSpecService.clusterChanges
+      .pipe(map(cluster => cluster.spec.clusterNetwork?.ipFamily))
+      .pipe(distinctUntilChanged())
+      .pipe(filter(ipFamily => !!ipFamily))
```

Comment:
Just for info! what is the purpose of filter operator are we using somewhere? I think we already have dual stack check on `_vpcListObservable`

Reply:
This was added to filter out all `null` or `undefined` values of `ipFamily` but now I think it should be removed.